integration-tests/src/bin/read.rs

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use core::ffi::c_void;
+use libc::read;
+
+fn main() {
+    let buf = [0u8; 12];
+    unsafe {
+        read(0, buf.as_ptr() as *mut c_void, 12);
+    }
+    print!("{:?}", String::from_utf8_lossy(&buf));
+}
```

Comment:
@jyotsna-penumaka I think what @npmccallum was looking for here was to read the input byte by byte (and output the same byte) until there are no more bytes (`EOF`), but I see this as just reading one byte. You would want a loop to keep reading input byte by byte and writing out the bytes. @npmccallum Let me know if I'm misunderstanding.

Reply:
@npmccallum I used break instead of exit system call, as the output would be 1 in that latter case, even though the input from stdin was read correctly.

Reply:
@jyotsna-penumaka Returning from `main()` is the same as `exit(0)`.